run_artemis.sh

```diff
@@ -163,7 +168,7 @@ if [[ $START == 1 ]]; then
 
     source .venv/bin/activate
 
-    python -m artemis `if [ $IN_DEV == true ]; then echo "--dev"; fi` >$start_log_path 2>&1 &
+    python -m artemis `if [ $IN_DEV == true ]; then echo "--dev"; fi` `if [ $SKIP_STARTUP_CONNECTION == 1 ]; then echo "--skip_startup_connection"; fi`>$start_log_path 2>&1 &
```

Comment:
This is ok for now but FYI I have added https://github.com/DiamondLightSource/python-artemis/issues/603 to tidy this